modules/landing_zone/main.tf

```
@@ -1,50 +1,50 @@
 resource "null_resource" "landing_zone_config" {
   triggers = {
-    command = "${var.landing_zone_command}"
+    command    = "${var.landing_zone_command}"
+    providers  = "${md5(jsonencode(var.landing_zone_providers))}"
     components = "${md5(jsonencode(var.landing_zone_components))}"
-    region = "${var.region}"
-    account_id = "${var.account_id}"
   }
 
   "provisioner" "local-exec" {
```

Argument names should not be quoted
```suggestion
   provisioner "local-exec" {
```